<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ContinueButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="SafetyEvents">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Building Safety Events:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Fire">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire </string>
    </property>
   </widget>
   <widget class="QPushButton" name="PowerOut">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="E1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="E2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevator 2:</string>
    </property>
   </widget>
   <widget class="QFrame" name="Elevator1">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="E1Open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open &lt;|&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E1Close">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close &gt;|&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E1Help">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E1Overload">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E1Obstacle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Door Obstacle</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="LogConsole">
    <property name="geometry">
     <rect>
      <x>607</x>
      <y>191</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Console</string>
    </property>
   </widget>
   <widget class="QLabel" name="P2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="P3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 3</string>
    </property>
   </widget>
   <widget class="QFrame" name="Passenger1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="P1StartFloor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Floor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination Floor</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="P1DestFloor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>At Time Step</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="P1Time">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="P1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Passenger 1</string>
    </property>
   </widget>
   <widget class="QFrame" name="Passenger3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="P3StartFloor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Floor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination Floor</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="P3DestFloor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>At Time Step</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="P3Time">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Passenger2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="P2StartFloor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Floor:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination Floor</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="P2DestFloor">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>At Time Step</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="P2Time">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Elevator2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="E2Open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open &lt;|&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E2Close">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close &gt;|&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E2Help">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E2Overload">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Overload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="E2Obstacle">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Door Obstacle</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="Log">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>491</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Time: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearLog">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Log Console</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetValues">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Values</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(Both elevators are at Floor 1 by default)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
